classes/person.rb

```diff
@@ -7,10 +7,11 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
   end
 
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rental
 
```

Comment:
- Hi, I would like to advise you to move this block of variables above `def initialize`. I am requesting this because we need to follow the ruby logic: **We define variables, then initialization follows**. 
![Capture](https://github.com/Progress2002/OOP-school-library/assets/63972670/7a58b422-8cff-4837-85e3-50cb9cdf5fbd)